qrisk/stats.py

```diff
@@ -314,10 +323,18 @@ def omega_ratio(returns, annual_return_threshhold=0.0):
 
     """
 
-    daily_return_thresh = pow(1 + annual_return_threshhold, 1 /
-                              APPROX_BDAYS_PER_YEAR) - 1
+    if returns.size < 2:
```

Comment:
I don't have a preference either way, but why do you think len() is better than .size?